docs/check-examples

```diff
@@ -379,11 +380,11 @@ do
         else
           if [ "$command" == "%check-legacy" ]; then
             isverbose=-compileVerbose:$checkIds
-            (cd "$dir"; "$dafny" $verb $dOptions $isverbose ${options[@]} $text   ) > "$actualOut" 2> "$actualError"
+            (cd "$dir"; "$dafny" $verb /showSnippets:0 $dOptions $isverbose ${options[@]} $text   ) > "$actualOut" 2> "$actualError"
```

Comment:
@keyboardDrummer  This looks suspicious, it should probably be --show-snippets:false

Reply:
$verb is always empty at this point -- this is the branch for legacy CLI tests. SO it is correct, but perhaps $verb could be removed.

Reply:
Removed $verb on this line to avoid confusion.

Reply:
And tests now pass after nightlies have been fixed.  Mikael - can you approve now?